udata/commands/fixtures.py

```diff
@@ -112,13 +124,14 @@ def generate_fixtures_file(data_source):
 
             json_result.append(json_fixture)
 
-    with open("results.json", "w") as f:
-        json.dump(json_result, f)
+    with results_file.open("w") as f:
+        json.dump(json_result, f, indent=2)
+        print(f"Fixtures saved to file {results_filename}")
 
 
 @cli.command()
 @click.argument("source", default=DEFAULT_FIXTURE_FILE)
-def generate_fixtures(source):
+def generate_fixtures(source: str) -> None:
```

Comment:
I think we may want to rename `generate_fixtures` by something like `import_fixtures` at some point

Reply:
Yes definitely! The current naming is confusing 😬 
I can change it right away if you think it still makes sense in the context of this PR

Reply:
Maybe in the refactoring PR instead? :)